util/io/bytereadseeker.go

```diff
@@ -0,0 +1,38 @@
+package io
+
+import (
+	"io"
+	"io/fs"
+)
+
+func NewByteReadSeeker(data []byte) *byteReadSeeker {
```

Comment:
Can't we just use https://pkg.go.dev/bytes#NewReader to do this?

Reply:
Nice find, I didn't know about this

Reply:
Thanks. I've updated with your suggestion